.vscode/tasks.json.sample

```diff
@@ -1,42 +1,73 @@
 {
-    "version": "2.0.0",
-    "tasks": [
-        {
-            "label": "Build",
-            "type": "shell",
-            "command": "mkdir -p build && cd build && cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_DEBUG_WASM=ON -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_INSTALL_PREFIX=\"psidk\" .. && make install",
-            "group": {
-                "kind": "build",
-                "isDefault": true
-            },
-            "options": {
-                "statusbar": {
-                    "label": "$(tools) Build"
-                }
-            }
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build",
+      "type": "shell",
+      "command": ".vscode/scripts/build.sh",
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "options": {
+        "statusbar": {
+          "label": "$(tools) Build"
         }
-        // Keeping the below example to show how to run service unit tests:
-        // {
-        //     "label": "Test NFT",
-        //     "type": "shell",
-        //     "command": "cd build && psitest Nft-test-debug.wasm -s -r compact",
-        //     "group": {
-        //         "kind": "test",
-        //         "isDefault": false
-        //     },
-        //     "options": {
-        //         "statusbar": {
-        //             "label": "$(beaker) Test Nft"
-        //         }
-        //     }
-        // },
-    ],
-    "inputs": [
-        {
-            "id": "testsuite",
-            "type": "promptString",
-            "default": "[mytestsuite]",
-            "description": "Enter test suite key (Example: \"[MyTestSuite]\""
+      }
+    },
+    {
+      "label": "Continue",
+      "type": "shell",
+      "command": ".vscode/scripts/continue.sh <YOUR_HOST_IP> myprod 8080",
```

Comment:
Is `<YOUR_HOST_IP>` something that needs to be replaced manually? Won't that cause a problem with psibase-contributor just copying the file?

Reply:
It must be replaced manually, yes. I'll have to add a note about this in the psibase contributor docs. Sounds crazy, but afaict there is no way to reliably be able to know what that IP will be in a generic way (across host OS).

Reply:
Didn't we find that it was 127.0.0.1 on some systems? Would it work to use add the ip address iff `$HOST_IP` is defined?

Reply:
I don't think it's ever 127.0.0.1.

On my windows/wsl2 setup it's currently `172.23.0.1` which can be queried from within the container using something like [this](https://github.com/gofractally/image-builders/blob/main/docker/psibase-contributor.Dockerfile#L82).

But on a Mac, I think http requests from the host are double-NATed, and nothing I try from within the container seems to be able to resolve to the IP address that we see in the psinode logs (Which is 192.168.65.1 or I think sometimes 192.168.65.2)

Reply:
Even so, I think it would be better to rely on an env variable instead of manually editing the config file. Is the address stable across stopping/starting the container or rebooting the host?

Reply:
Okay, I figured out a way to be able to continue to use $HOST_IP here. The drawback is that I need mac-os specific logic in `psibase-contributor`.